R/utils-limits.R

```diff
@@ -82,8 +82,7 @@ testCalloutsByView <- function(plot_obj, view_num, xlimits_real, width_char, xra
     callout_args <- plot_obj[[view_num]][[i]]
     
     if (!isEmptyOrBlank(callout_args$x)) {  
-      text_len <- nchar(callout_args$labels)
-      
+      text_len <- ifelse(is.na(callout_args$labels),0,nchar(callout_args$labels))
```

Comment:
don't need this to check `isEmptyOrBlank(callout_args$labels)` here?

Reply:
I guess depends on if the value you are trying to catch is `NA` or `NULL`

Reply:
We're trying to catch NA, but what I found is that isEmptyOrBlank only returns true if the entire vector is empty or blank.

Reply:
Ahhhh I wonder if we should create another method of isEmptyOrBlank for vectors...what else is it not catching that it should?